tests/stdlib/test_math.py

```diff
@@ -0,0 +1,23 @@
+from ..utils import TranspileTestCase
+
+
+class MathModuleTests(TranspileTestCase):
+
+    def test_sqrt(self):
+        def create_test(arg):
+            return f"""
+                import math
+                try:
+                    print(math.sqrt({arg}))
+                except Exception as e:
+                    print(type(e), ':', e)
+            """
+
```

Comment:
Please refrain from making a single test to test for every functionality. It makes it a lot harder to determine what is not functioning properly when the only thing the unittests will show is that "test_sqrt" failed or succeeded. Instead create separate tests for all of type of tests created below.

Reply:
Fixed by fc1917f8a717a980507ed8a2ada8f6b0b7753b98

Reply:
Agreed!